src/main/java/com/github/martynagil/storemanagement/UserInterface.java

```diff
@@ -11,7 +17,65 @@ public UserInterface(Console console, ProductService productService) {
         this.productService = productService;
     }
 
-    public void exit() {
+    public void mShowProducts() {
+        showProducts(productService.getAllProducts());
+    }
+
+    public void mAddProduct() {
+        String name = console.askForString("Enter the name of product: ");
+        String brand = console.askForString("Enter the name of brand: ");
+        String type = console.askForString("Enter the type: ");
+        String barcode = console.askForString("Enter the barcode: ");
+        double price = console.askForDouble("Enter the price: ");
```

Comment:
jeżeli chodzi o obsługę błędów, to co jak ktoś poda w tym miejscu np. `asasd`? Jak myślisz, gdzie obsługa tego przypadku powinna się znaleźć?

Reply:
Powinno się pokazywać, aż się wpisze dobrze. No to chyba tutaj powinna być ta obsługa, ale nie wiem jak. Pętlą, że dopóki rzuca bład to się ma pojawiać?

Reply:
no generalnie tak, ale powinno być raczej w `Console` i pytamy dopóki nie uzyskamy double skoro o niego prosiliśmy